main.py

```diff
@@ -24,7 +24,7 @@
 )
 
 
-DATASET = os.environ.get("BIGQUERY_DATASET")
+DATASETS = os.environ.get("BIGQUERY_DATASET").split(os.pathsep)
```

Comment:
so they're specified with a slash?

Reply:
ah, a `:`, maybe make this explicit? obviously `os.pathsep` is going to be the same across posix stuff, but this could potentially be bewildering at some point.

Reply:
Opted to split on whitespace instead since these dataset names won't have whitespace, and `.strip()` will return an empty list when given an empty string.